src/patternfly/components/Table/table-favorite-sort.hbs

```diff
@@ -0,0 +1,18 @@
+{{#> action-list action-list--modifier='pf-m-icons'}}
+  {{#>button
```

Comment:
Is this missing the action-list-item component around the buttons?

Reply:
EagleEyeBlindGuy 😆 I'll add `.action-list__item`

Reply:
🦅